### ISSUE TITLE
[CKEditor 5] Upcasting strips all valid MathML attributes

### ISSUE THREAD

**Issue by icc** [wiris/html-integrations#881]
## Description

I belive the following change should have used ADD_ATTRS as opposed to ALLOWED_ATTR:
https://github.com/wiris/html-integrations/commit/1615d3a8ac8146ea82b3a195e74ce059f8c1dd59#diff-e73193ebff72abb4a904eea7beffdaf505ebc7628d9d824c772aa85eaeff42b4R407
as this will strip a lot of attributes officially supported in MathML.

When inserting via the WIRIS MathType editor the MathML contains all the right attributes. 
When opening the CKEditor again after saving, the upcasting triggers the DOMPurifier that removes all of the attributes.
This is a big problem as it changes the content without the author's awareness when editing existing content.

In addition, the MathML without the attributes are not compatible with the WIRIS service. 

## Example

Created the following using WIRIS MathType editor inside CKEditor:
```
<math xmlns="http://www.w3.org/1998/Math/MathML">
	<msub>
		<mi>a</mi>
		<mn>0</mn>
	</msub>
	<mo>+</mo>
	<mfrac>
		<mn>1</mn>
		<mrow>
			<msub>
				<mi>a</mi>
				<mn>1</mn>
			</msub>
			<mo>+</mo>
			<mstyle displaystyle="true">
				<mfrac>
					<mn>1</mn>
					<mrow>
						<mo>.</mo>
						<mo>.</mo>
						<mo>.</mo>
					</mrow>
				</mfrac>
			</mstyle>
		</mrow>
	</mfrac>
</math>
```

After second edit and save (without changing anything) looks like this:
```
<math>
	<msub>
		<mi>a</mi>
		<mn>0</mn>
	</msub>
	<mo>+</mo>
	<mfrac>
		<mn>1</mn>
		<mrow>
			<msub>
				<mi>a</mi>
				<mn>1</mn>
			</msub>
			<mo>+</mo>
			<mstyle>
				<mfrac>
					<mn>1</mn>
					<mrow>
						<mo>.</mo>
						<mo>.</mo>
						<mo>.</mo>
					</mrow>
				</mfrac>
			</mstyle>
		</mrow>
	</mfrac>
</math>
```

If you try POSTing this to https://www.wiris.net/demo/plugins/app/service :
```
{
  "service": "mathml2latex",
  "mml": "<math><msub><mi>a</mi><mn>0</mn></msub><mo>+</mo><mfrac><mn>1</mn><mrow><msub><mi>a</mi><mn>1</mn></msub><mo>+</mo><mstyle><mfrac><mn>1</mn><mrow><mo>.</mo><mo>.</mo><mo>.</mo></mrow></mfrac></mstyle></mrow></mfrac></math>"
}
```
you get the following response: 
```
{"status":"error","errors":["Error converting from MathML to LaTeX"]}
```
This is because of the missing `displaystyle="true"` attribute.

**Comment by carla-at-wiris** [comment]
Hi @icc, thank you for the detailed report and the suggested changes,

Our team took a look at it and will send your proposal to the next step. Once it's ready to release, we'll let you know! Any further issues, please feel free to contact us

**Comment by carla-at-wiris** [comment]
Hello @icc,

We released a new version `8.8.3` with the changes that you suggested. Any further issues, please feel free to contact us

**Comment by icc** [comment]
Super 😍 thank you!
